src/containers/ServiceContainers.js

```diff
@@ -3,19 +3,39 @@ import { connect } from 'react-redux';
 import { CreateOrUpdate, List, Detail } from '../components/Service';
 
 
+function mergeServices(state) {
```

Comment:
this could be written as `function mergeServices({ service, area, category}) { ... }`